src/deadline/client/cli/mcp_server.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+The `deadline mcp-server` command.
+"""
+
+import sys
+import click
+
+from ._common import _handle_error
+
+
+@click.command(name="mcp-server")
+@_handle_error
+def cli_mcp_server():
```

Comment:
Do we get telemetry automatically whenever this command is run? It'd be interesting to know how often the MCP server is started, even if the individual tools aren't called. Optional and not blocking.